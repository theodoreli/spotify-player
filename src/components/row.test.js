import React from 'react';
import Row from './row.js';
import renderer from 'react-test-renderer';

import singleRes from '../mock-data/single-response-object.js';

test('Row renders as expected', () => {
  const component = renderer.create(
    <Row rowData={singleRes}/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
