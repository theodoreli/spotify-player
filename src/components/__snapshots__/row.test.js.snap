// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders as expected 1`] = `
<li
  className="Row-listElement-0-1-5"
>
  <div
    className="Row-rowMainInfo-0-1-6"
  >
    <svg
      aria-hidden="true"
      className="Row-issueStatus-0-1-3"
      height="16"
      viewBox="0 0 14 16"
      width="14"
    >
      <path
        d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"
      />
    </svg>
    <div
      className="rowMainText"
    >
      <div
        className="Row-issueName-0-1-2"
      >
        <a
          href="https://github.com/tensorflow/tensorflow/pull/18838"
        >
          Add DeviceSet to Cluster
        </a>
         
        <span
          className="Row-label-0-1-4"
          style={
            Object {
              "backgroundColor": "#5319e7",
            }
          }
        >
          stat:awaiting response
        </span>
      </div>
      <div
        className="Row-issueDetails-0-1-1"
      >
        #
        317395896
         opened on 
        Fri Apr 20 2018
         by 
        aaroey
      </div>
    </div>
  </div>
  <div
    className="rowSideInfo"
  >
    <img
      alt="User avatar"
      className="Row-userAvatar-0-1-7"
      src="https://avatars1.githubusercontent.com/u/326106?v=4"
    />
  </div>
</li>
`;
